pkg/translator/prometheus/normalize_name.go

```diff
@@ -164,7 +164,7 @@ func normalizeName(metric pmetric.Metric, namespace string) string {
 
 	// Append _total for Counters
 	if metric.DataType() == pmetric.MetricDataTypeSum && metric.Sum().IsMonotonic() {
-		nameTokens = append(removeItem(nameTokens, "total"), "total")
+		nameTokens = appendSuffixIfMissing(nameTokens, "total")
```

Comment:
We need to make sure the word "total" doesn't appear twice, so we remove it and then append it.

Otel cumulative sum `network.io.total` with units `By` will be transformed following this workflow:

1. slice ==> `["network", "io", "total"]`
2. append the unit ==> `["network", "io", "total", "bytes"]`
3. remove "total", and then append it again ==> `["network", "io", "bytes", "total"]`
4. join ==> `network_io_bytes_total`

Using *appendSuffixIfMissing()* we would get `network_io_total_bytes_total` which is not desirable.

Please revert (or detail the case you're trying to cover), thank you!